neurobooth_os/server_stm.bat

```diff
@@ -1,4 +1,3 @@
-call C:\Users\STM\anaconda3\Scripts\activate.bat %NB_STM_CONDA%
+call C:\Users\STM\anaconda3\Scripts\activate.bat %NB_CONDA%
 start /W ipython --pdb %NB_INSTALL%\neurobooth_os\server_stm.py
```

Comment:
Should this be call start? instead of just start? The other server bats have call start and I am wondering if this is different for STM for a specific reason.

Reply:
Was actually wondering myself why the other scripts use `call start`. The `call` seems redundant in that pairing.

Reply:
(As for removing the `call` from other scripts, maybe we leave them be for now and just fix when we touch those scripts for other reasons.)

Reply:
I agree with Brandon on this. I also dont know why we use call start, and from the documentation I read this seemed redundant. Let's remove this, and if we notice odd behavior we can add this back in with a comment.

Reply:
just to be clear, are we saying that all of the call starts in all three bat files should be changed to just start? There are two or three in each file

Reply:
If it's just a few scripts (e.g., server start scripts) I think we can pull the trigger. If it's a lot of files then we're potentially exposing ourselves to risk from several angles if there *is* a reason for it. Maybe we just keep a list of affected files and keep an eye out for potentially weird bugs or odd behavior.

Reply:
It's just the start scripts
I will make the change (and fix one typo in one of the scripts.